pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -626,3 +627,20 @@ func (c *Cluster) getExpandPVCInitContainer(osdProps osdProperties, osdID string
 
 	return container
 }
+
+func getPauseContainerCommands(command []string, args []string) []string {
+	var pauseCommandArgs = []string{
```

Comment:
Instead of using individual args, how about a single script in the same pattern that the [toolbox job example](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/toolbox-job.yaml#L41-L48) uses?

Reply:
So @travisn are you suggesting just have this kind of behaviour where we run `bash -c  ceph osd....` and the developer could go and this command to `sleep infinity` ?

Reply:
I like the approach of the env var `PAUSE_OSD` so the admin just has to modify that var if they want to pause it. I don't expect they should modify a script, though the script would certainly be more flexible that way if they wanted to run some other command besides just pausing. My point was just that the script would be better for code readability. At the same time, it's not like these args are hard to read, so you can leave the args separate like this too. We just need to move it to an init container.